config/locales/errors/en.yml

```diff
@@ -35,6 +35,9 @@ en:
         %{timeout}.</strong>'
       send_link_limited: You tried too many times, please try again in %{timeout}. You
         can also go back and choose to use your computer instead.
+    enter_code:
+      rate_limited_html: You entered an incorrect verification code too many
+        times.<br> <strong>Try again in %{timeout}.</strong>
```

Comment:
Typically I'd think we'd avoid `<br>` line breaks because they cause weird unexpected line breaks at different screen sizes. Are we expecting that for mobile displays?

<img src="https://github.com/18F/identity-idp/assets/1779930/2e174dde-e152-4838-81aa-c543c8c626f2" width="280" alt="screenshot">

Reply:
I'll ask how it is supposed to be viewed on smaller screens.